app/views/homepage/index.haml

```diff
@@ -124,7 +128,7 @@
           - if @view_type.eql?("grid")
             %div{class: @category_menu_enabled ? "home-fluid-thumbnail-grid-narrow" : "home-fluid-thumbnail-grid-wide"}
               .home-fluid-thumbnail-grid
-                = render :partial => "grid_item", :collection => @listings, :as => :listing, locals:{show_distance: show_distance}
+                = render :partial => "grid_item", :collection => @listings, :as => :listing, locals:{show_distance: location_search_in_use}
```

Comment:
"locals:{" is changed to "locals: {" in another branch which probably goes to master first (so, some conflicts ahead)